### PR TITLE
frontend: making BitBoxApp logo link to '/'

### PR DESCRIPTION
**Pull request by jstrnbrg**

### CODE REVIEW

**Comment by jstrnbrg** [comment]
Can't reproduce, works for me :(

**Comment by Tomasvrba** [comment]
> Can't reproduce, works for me :(

My bad, not enough description. It works with servewallet, but doesn't work with the built Qt app. I expect it has something to do with the full page reload that you see in servewallet

File: frontends/web/src/components/sidebar/sidebar.tsx [hunk 1]
```diff
@@ -165,9 +165,14 @@ class Sidebar extends Component<Props> {
             <div className={['sidebarContainer', hidden ? 'forceHide' : ''].join(' ')}>
                 <div className={['sidebarOverlay', activeSidebar ? 'active' : ''].join(' ')} onClick={toggleSidebar}></div>
                 <nav className={['sidebar', activeSidebar ? 'forceShow' : '', shown ? 'withGuide' : ''].join(' ')}>
-                    <div className="sidebarLogoContainer">
-                        <AppLogoInverted className="sidebarLogo" />
-                    </div>
+                    <Link
```
Comment (benma) [inline frontends/web/src/components/sidebar/sidebar.tsx]: is Link the right component? (don't know what it does). could be `<a ...>` as well?

@originalblend337

Comment (thisconnect) [reply frontends/web/src/components/sidebar/sidebar.tsx]: Link is fine

File: frontends/web/src/components/sidebar/sidebar.tsx [hunk 2]
```diff
@@ -165,9 +165,14 @@ class Sidebar extends Component<Props> {
...
+                        activeClassName=""
+                        href={`/`}
```
Comment (thisconnect) [inline frontends/web/src/components/sidebar/sidebar.tsx]: `href="/"`
nit: would be enough

**Comment by thisconnect** [comment]
This was merged  https://github.com/digitalbitbox/bitbox-wallet-app/blame/382e57b08079e0773107ca7e3f85db806cd60101/frontends/web/src/components/sidebar/sidebar.tsx#L168-L175

close PR?

**Comment by Tomasvrba** [comment]
> This was merged https://github.com/digitalbitbox/bitbox-wallet-app/blame/382e57b08079e0773107ca7e3f85db806cd60101/frontends/web/src/components/sidebar/sidebar.tsx#L168-L175
> 
> close PR?

It wasn't, this is master: https://github.com/digitalbitbox/bitbox-wallet-app/blob/5d83d4ff1b85dedf39697a2761fbafe46f1c9aa7/frontends/web/src/components/sidebar/sidebar.tsx#L168

File: frontends/web/src/components/sidebar/sidebar.tsx (hunk 2 above)
Comment (jstrnbrg) [reply frontends/web/src/components/sidebar/sidebar.tsx:170]: done

File: frontends/web/src/components/sidebar/sidebar.tsx (hunk 1 above)
Comment (jstrnbrg) [reply frontends/web/src/components/sidebar/sidebar.tsx:168]: So should I use `Link` or `<a>` or something else?

File: frontends/web/src/components/sidebar/sidebar.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/components/sidebar/sidebar.tsx:170]: did you push?

File: frontends/web/src/components/sidebar/sidebar.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/components/sidebar/sidebar.tsx:168]: Pleaes keep using the Link component from preact-router.

**Pull request closed without merging**